### PR TITLE
Merge user creation api changes to User_Creation

### PR DESCRIPTION
**Pull request by mayuri0723**

### CODE REVIEW

File: backend/services/userService.js [hunk 1]
```diff
@@ -259,11 +259,12 @@ const createPersonalDetails = async (req, res) => {
     // Add educational details field
     const educationalDetails = await EducationDetails.create(req.body.educationalDetails);
     personalDetails.educationalDetails = educationalDetails.id;
-    // Add bank details field
+    // // Add bank details field
```
Comment (ShubhamShete24) [inline backend/services/userService.js:262]: remove extra //

File: backend/services/userService.js [hunk 2]
```diff
@@ -259,11 +259,12 @@ const createPersonalDetails = async (req, res) => {
...
     const bankDetails = await BankDetails.create(req.body.bankDetails);
     personalDetails.bankDetails = bankDetails.id;
 
     const newPersonalDetails = await PersonalDetails.create(personalDetails);
+    console.log('personalDeatilsbody', personalDetails);
```
Comment (ShubhamShete24) [inline backend/services/userService.js]: remove console

File: backend/routers/userRouter.js [hunk 3]
```diff
@@ -17,7 +18,8 @@ userRouter.post('/authenticate', authenticate);
 userRouter.put('/update-user-info', verifyToken, updateUserInfo);
 userRouter.post('/assign-role', verifyAdmin, assignRole);
 userRouter.post('/assign-manager', verifyAdmin, assignManager);
-userRouter.get('/get-users', verifyAdmin, getUsers);
+userRouter.get('/get-users', getUsers);
 userRouter.post('/create-personalDetails', createPersonalDetails);
+userRouter.post('/create-employeeDetails', createEmployeeDetails);
```
Comment (Arunangshu37) [inline backend/routers/userRouter.js:23]: If model name is EmploymentDetails won't it be better to call `create-employeeDetails` to `create-employmentDetails`

File: backend/services/userService.js [hunk 4]
```diff
@@ -250,4 +290,41 @@ const createPersonalDetails = async (req, res) => {
   }
 };
 
-export { createUser, authenticate, updateUserInfo, assignRole, assignManager, getUsers, createPersonalDetails };
+const createEmployeeDetails = async (req, res) => {
```
Comment (Arunangshu37) [inline backend/services/userService.js:293]: If model name is EmploymentDetails won't it be better to call `createEmployeeDetails` to `createEmploymentDetails`.

**Pull request merged**